### PR TITLE
remove destructive and warning button type

### PR DESCRIPTION
**Pull request by ElaLukaszewicz**
remove destructive and warning button type (https://jira.z-dn.net/browse/ANSPRIO-2965)

### CODE REVIEW

File: src/components/buttons/README.md [hunk 1]
```diff
@@ -87,19 +87,11 @@ import {Button} from 'style-guide';
 ```
 
 #### Additional buttons
-There are also 3 additional buttons created for special cases - destructive and warning, and facebook button, what have custom set of colors from outside the palette.
+There are also 1 additional button created for special case: facebook button, which has custom color from outside the palette.
```
Comment (matzimowski) [inline src/components/buttons/README.md:90]: "There is"

**Comment by seahindeniz** [comment]
@ElaKrk please see https://github.com/brainly/style-guide/issues/1584#issuecomment-509237104

**Comment by matzimowski** [comment]
Hi @seahindeniz,
thanks for being alert 👌 
Could you please use `link-button-peach` and `link-button-mustard` instead the ones we are removing?
It's a changed requested by our design team, they are not planning to use them anymore in our projects, so there's no reason for us to maintain them and we have to stay in sync with our Design System. too
We are planning to introduce further changes to buttons soon, maybe some new types will appear.

**Comment by seahindeniz** [comment]
Hi @matzimowski 

Thanks for letting me know and I always in wait for new plans, changes.
But I just want you to remind that, this project is not going to be used only on the Brainly web app or Brainly's sub-projects. So please make changes that would not affect others.

Also I can't open link in description but I would like to ask; is it absolutely necessary to remove colors from buttons?
I'm already using the `link-button-peach, along with `sg-button--destructive` and both are represents different things. I would like to ask changing names to `primary-peach` and `primary-mustard` to comply the naming standards in Colors- instead of `destructive` or `warning` before, but I didn't bother because either way buttons would be exists.

**Comment by seahindeniz** [comment]
Anyways :) 
I don't want to get in your way if it's one of the job requirements. So I hope design team would consider extending additional colors on the primary buttons.
Thanks!

**Comment by matzimowski** [comment]
@seahindeniz 
thanks for your understanding 👌 

I see your point, but unfortunately, in case someone still wants to use those types of buttons, they need to stick to a particular version of style-guide.

changes that we gonna do are not public yet from design perspective and i am not sure when exactly we introduce them, but yea, that's our plan.

**Comment by seahindeniz** [comment]
@matzimowski the version topic from my part has already been discussed in here https://github.com/brainly/style-guide/issues/1584#issuecomment-498733034

> For the newer UI, my extension uses the components of the style-guide.css that loaded in DOM. That means, extension doesn't have a control of <style /> tag of the style-guide.css file in the <head /> tag added by Brainly.

If you make changes, that means I get affected as well and I have to comply accordingly so that is why I'm tried to explain. Please understand my concerns too :slightly_smiling_face:

**Comment by ElaLukaszewicz** [comment]
@seahindeniz I will let you know when we will release new version of SG. I believe it will happen on Monday or Tuesday next week.

**Comment by matzimowski** [comment]
@seahindeniz yea, I see your point.

Actually, regarding these buttons we removed, it's just a removal of modifiers, so they will both fallback to `sg-button--primary` and become "dark" ones. 🕶 

I guess it breaks your UI, but yea, for now what we can suggest is just to switch to `sg-button--link-button-peach` and `sg-button--link-button-mustard`.

thanks again and sorry for inconvenience.

**Comment by seahindeniz** [comment]
@ElaKrk got it, thanks :wink: 

@matzimowski I'm trying not to create customized styles from my side but that's OK. I can manage somehow for having the `primary-peach` and `primary-mustard` modifiers exists :rocket:

**Pull request merged**